### DISCUSSION TITLE
Scratch Deskop

### DISCUSSION THREAD

**Discussion by Thomas282-coder** [ScratchAddons/ScratchAddons#1103] [Help] [answered]
Can we use this add-on in Scratch Desktop, or will you add it?

**Comment by Marc92020** [comment]
I'm not really sure, but if they did, Scratch Addons would have to be an app too.

**Reply by Marc92020** [reply]
but I've tested it out and no addons work.

**Comment by GrahamSH-LLK** [comment]
It's impossible

**Reply by WorldLanguages** [reply]
Well, I guess we could fork Scratch Electron...

**Reply by GrahamSH-LLK** [reply]
Yeah, no

**Comment by Marc92020** [comment]
There would probably have to be an app

**Reply by GrahamSH-LLK** [reply]
No, it woudn't work in general. We would have to port the entire codebase into the scratch electron app, and it's honestly not worth it. You also couldn't use the actual scratch desktop, or get it from the stores, etc.

**Comment by apple502j** [comment]
This is rejected.

**Comment by WorldLanguages** [comment, marked as answer]
Well, you can now use most editor addons in TurboWarp desktop: https://desktop.turbowarp.org/

**Comment by ghost** [comment]
No, it would require literally making an entire copy of the desktop app and adding addons to it. So SA would basically be an app.

**Reply by GrahamSH-LLK** [reply]
This has already been freaking answered.

**Comment by gazalied** [comment]
Closed as Resolved?

**Reply by retronbv** [reply]
You cannot close discussions

**Reply by Hans5958** [reply]
@retronbv Closed
